server/flaskr/services/notifications/notification_service.py

```
@@ -0,0 +1,55 @@
+import time
+
+import firebase_admin
+from firebase_admin import credentials, messaging
+
+from .handlers import handlers
+from flaskr.database.handlers import NotificationJobDataHandler
+
+class NotificationService:
+    @classmethod
+    def init(cls):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `init`

Add return type annotation: `None`

(ANN206)